sfdc37/salesforce_utils.bal

```diff
@@ -138,8 +138,7 @@ function checkAndSetErrors(http:Response|error response, boolean expectPayload)
         error err => {
             SalesforceConnectorError connectorError =
             {
```

Comment:
we normally use Egyptian style brackets, which is opening bracket fronted with only a single space and closing bracket is align with the parent line start, hence let's reformat this to be as below ? this is just a suggestion.
`SalesforceConnectorError connectorError = {`

Reply:
Fixed.